tests/jsmarionette/client/marionette-client/lib/marionette/client.js

```diff
@@ -938,8 +879,8 @@
       }
 
       setState(this, 'context', context);
-      var cmd = { name: 'setContext', parameters: { value: context }};
-      return this._sendCommand(cmd, 'ok', callback);
+      var cmd = {name: 'setContext', parameters: {value: context}};
```

Comment:
Any reason this line changed? Not sure this passes the lint check after the change